src/Mvc/test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/ValidationSummaryTagHelperTest.cs

```diff
@@ -587,7 +588,70 @@ public async Task ProcessAsync_GeneratesValidationSummaryWhenNotNone(ValidationS
                 expectedMessage);
         }
 
-        private static ViewContext CreateViewContext()
+        [Fact]
+        public async Task ProcessAsync_GeneratesExpectedOutput_WithModelErrorForIEnumerable()
+        {
+            // Arrange
+            var expectedError = "Something went wrong.";
+            var expectedTagName = "not-div";
+            var expectedAttributes = new TagHelperAttributeList
+            {
+                new TagHelperAttribute("class", "form-control validation-summary-errors"),
+                new TagHelperAttribute("data-valmsg-summary", "true"),
+            };
+
+            var metadataProvider = new TestModelMetadataProvider();
+            var htmlGenerator = new TestableHtmlGenerator(metadataProvider);
+            var validationSummaryTagHelper = new ValidationSummaryTagHelper(htmlGenerator)
+            {
+                ValidationSummary = ValidationSummary.All,
+            };
+
+            var expectedPreContent = "original pre-content";
+            var expectedContent = "original content";
+            var tagHelperContext = new TagHelperContext(
+                tagName: "not-div",
+                allAttributes: new TagHelperAttributeList(),
+                items: new Dictionary<object, object>(),
+                uniqueId: "test");
+            var output = new TagHelperOutput(
+                expectedTagName,
+                attributes: new TagHelperAttributeList
+                {
+                    { "class", "form-control" }
+                },
+                getChildContentAsync: (useCachedResult, encoder) =>
+                {
+                    var tagHelperContent = new DefaultTagHelperContent();
+                    tagHelperContent.SetContent("Something");
+                    return Task.FromResult<TagHelperContent>(tagHelperContent);
+                });
+            output.PreContent.SetContent(expectedPreContent);
+            output.Content.SetContent(expectedContent);
+            output.PostContent.SetContent("Custom Content");
+
+            var model = new FormMetadata();
+            var viewContext = TestableHtmlGenerator.GetViewContext(model, htmlGenerator, metadataProvider);
+            validationSummaryTagHelper.ViewContext = viewContext;
+
+            var modelState = viewContext.ModelState;
+            SetValidModelState(modelState);
```

Comment:
Remove this call. The tag helper doesn't look at the values in ModelState, just the errors.